Tests/AreaTest.cs

```diff
@@ -0,0 +1,106 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Transactions;
+using ExotischNederland.Models;
+using System.Collections.Generic;
+
+namespace Tests
+{
+    [TestClass]
+    public class AreaTest : TransactionTest
+    {
+        [TestMethod]
+        public void TestAreaCanBeCreated()
+        {
+            string name = "Test Area";
+            string description = "A beautiful natural area for testing.";
+            string polygonPoints = "POINT(1 1), POINT(2 2), POINT(3 3)";
```

Comment:
Is this how points are defined, or just as a test?

We should define the points and then change the model and test for the string <-> array functionality